<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GPIOTab</class>
 <widget class="QWidget" name="GPIOTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>258</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTimeEdit" name="te_refresh_time">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>231</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">::up-button, ::down-button {
    width:40px;
    height:40px;
}

QTimeEdit {

font-size:22px;

}</string>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>1</second>
     <year>2000</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::MinuteSection</enum>
   </property>
   <property name="displayFormat">
    <string>mm:ss.zzz</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_input">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>300</width>
     <height>35</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>INPUT GPIO  PI0 (256)</string>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_refresh">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>350</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh time:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_output">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>300</width>
     <height>35</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>OUTPUT GPIO PI1 (257)</string>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_na">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>15</y>
     <width>571</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:60px;
color:red;</string>
   </property>
   <property name="text">
    <string>NOT AVAILABLE</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
